### PR TITLE
initial translation proposal for c-1 and c-2

### PR DESCRIPTION
**Pull request by GenericUser394753**

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/57/0/translation.txt [hunk 1]
```diff
@@ -2,9 +2,11 @@
 // Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
 //
 フュル=Fur
-//当店のデラックスプランをご購入頂き、誠にありがとうございます。=
-//こちら、お客様へよりよいリビドーをご提供させて頂く為の特別なサービスとなっております。=
-//ただし、こちらはお客様との仲を深めたパートナーのみ受けられるものとなっております。=
-//そういったパートナーがいらっしゃらない場合はご遠慮させて頂く形になる事、ご了承下さい。=
-//当プランをお受けしたい場合は、改めて私に声をおかけ下さいませ。=
-//それでは、失礼致します。=
+当店のデラックスプランをご購入頂き、誠にありがとうございます。= Thank you for purchasing our Deluxe Plan.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/57/0/translation.txt:5]: In the game menus, Deluxe Plan was translated as Deluxe Service, its a good idea to keep the standard.
Also, never use space after the "=" sign. The translation must start immediately.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
-//私もお手伝いに来たんです……うふふ。=
-//お手伝い……ですか？=
-//はい……そちらのお客様とするなら、私も混ぜて頂こうと思いまして。=
-//先輩がですか……なるほど、私は特に問題はありません。=
-//そうですか、良かったです……それでは早速、ご一緒させていただきますね。=
+先輩……？　どうしてこちらに。= Miss Sitri...? Why are you here?
+私もお手伝いに来たんです……うふふ。= I came here to give you a hand as well... Hehehe.
+お手伝い……ですか？= Give a hand, you say?
+はい……そちらのお客様とするなら、私も混ぜて頂こうと思いまして。= Yes... if it's with the esteemed customer, I thought I'd like to join in too. 
+//はい……そちらのお客様とするなら、私も混ぜて頂こうと思いまして。= Yes... if it's with the esteemed customer, I thought I'd like to take part as well. (version 2)
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt:11]: Don't forget the capitalization after a dot. 
Wouldn't be better "Yes... If **you are** with the esteemed customer....."?

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt [hunk 3]
```diff
@@ -2,14 +2,17 @@
 // Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
 //
 フュル=Fur
-//……お疲れ様でした。=
+//……お疲れ様でした。= ...You've been very helpful.
+//……お疲れ様でした。= ...Thank you for your hard work. (technically, the most accurate one, but too many thanks going around, and usually it's the superior that would say this, so not fur)
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt:6]: No need to make alternative translations, just make your best shot. Revisions are here for these things^^

**Review by SpockBauru — CHANGES_REQUESTED** [review]
Watch out the punctuation, also don't use space after a "=" sign.
Also there's no need to make alternative translations, just give your best shot, revisions are here for these things ^-^
Overall seems good to me, but I'll need help from other reviewers...

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt [hunk 4]
```diff
@@ -2,11 +2,13 @@
 // Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
 //
 シトリー=Sitri
-//……少々お待ちください。=
+……少々お待ちください。= ... Please wait a moment.
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt:5]:
```suggestion
……少々お待ちください。=...Please wait a moment.
```
No space between leading `...` and words

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt (hunk 2 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt:11]: `I thought` => `I think`

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt (hunk 3 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt:6]: You can alway comment on your own review with alternate suggestions if there's places you can't decide with the

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt (hunk 2 above)
Comment (GenericUser394753) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/0/translation.txt:11]: agreed!

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt [hunk 5]
```diff
@@ -2,14 +2,17 @@
...
-//ありがとうございます、フュルさん……お手伝いさせてもらって私も嬉しかったです。=
-//こちらこそ、ありがとうございました。=
-//もしよかったら、今後もお手伝いに来てもよろしいでしょうか。=
-//私は、特に構いませんが……=
-//うふふ、それは良かったです。=
-//では、今度先輩がお客様とご一緒する時は、私がお手伝いに伺います。=
-//それは名案ですね……私もそうしてもらえれば心強いです。=
-//それでは、今後とも……=
-//えぇ、今後ともずっと一緒に……よろしくお願い致します。=
+ありがとうございます、フュルさん……お手伝いさせてもらって私も嬉しかったです。= Thanks, Fur... I was happy to help.
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt:10]: I think having her say `Miss Fur` would sound more in character

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/3/translation.txt [hunk 6]
```diff
@@ -3,5 +3,6 @@
 //
 //事後{0}=
 シトリー=Sitri
-//ふぅ……さすがに、少々疲れてしまいました……=
-//お客様……大変気持ち良いのですが、もう少々、加減をしていただけると助かります……=
+ふぅ……さすがに、少々疲れてしまいました……=Phew.... As I expected, this tired me out a little.
+//お客様……大変気持ち良いのですが、もう少々、加減をしていただけると助かります……=Dear customer, it felt very good but it would be very helpful if you could stop it for a little while. 
+お客様……大変気持ち良いのですが、もう少々、加減をしていただけると助かります……=Dear customer, it felt very good but it would be very helpful if we could take it slow for a little while.
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/3/translation.txt:8]:
```suggestion
お客様……大変気持ち良いのですが、もう少々、加減をしていただけると助かります……=Dear customer... It feels nice, but I would appreciate it if you slowed it down a bit.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/4/translation.txt [hunk 7]
```diff
@@ -3,5 +3,5 @@
 //
 //事後{0}=
 シトリー=Sitri
-//少々戯れたかっただけなのですか？　しようがないお客様ですね……=
-//困ったお客様です、あまり時間を取らせないで下さいね？=
+少々戯れたかっただけなのですか？　しようがないお客様ですね……= So you just wanted to have a little fun? You're helpless, aren't you...
+困ったお客様です、あまり時間を取らせないで下さいね？= You're such a troubling customer, please don't take up too much time, allright?
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/4/translation.txt:7]:
```suggestion
困ったお客様です、あまり時間を取らせないで下さいね？=You are a difficult customer, please don't take too long, okay?
```

Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-2/50/56/4/translation.txt:7]: `You're` would be fine/better, oops.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt [hunk 8]
```diff
@@ -2,17 +2,14 @@
...
-もしよかったら、今後もお手伝いに来てもよろしいでしょうか。= If it's allright, i can come help you in the future as well. 
-私は、特に構いませんが……= I wouldn't mind that...
-うふふ、それは良かったです。= Hehehe, I'm glad. 
-では、今度先輩がお客様とご一緒する時は、私がお手伝いに伺います。= In that case, the next time you are with the esteemed customer, i can come assist you as well.
-それは名案ですね……私もそうしてもらえれば心強いです。= That's a good idea... I would be glad if you could do the same.
-それでは、今後とも……= Well then, from now on...
-えぇ、今後ともずっと一緒に……よろしくお願い致します。= Yeah, from now on we'll be working together. I'm looking forward to it!
+ありがとうございます、フュルさん……お手伝いさせてもらって私も嬉しかったです。=Thanks, Miss Fur... I was happy to help. 
+こちらこそ、ありがとうございました。=You have my thanks as well.
+もしよかったら、今後もお手伝いに来てもよろしいでしょうか。=If it's allright, i can come help you in the future as well.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt:9]: i can > I can

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt [hunk 9]
```diff
@@ -2,17 +2,14 @@
...
-では、今度先輩がお客様とご一緒する時は、私がお手伝いに伺います。= In that case, the next time you are with the esteemed customer, i can come assist you as well.
-それは名案ですね……私もそうしてもらえれば心強いです。= That's a good idea... I would be glad if you could do the same.
-それでは、今後とも……= Well then, from now on...
-えぇ、今後ともずっと一緒に……よろしくお願い致します。= Yeah, from now on we'll be working together. I'm looking forward to it!
+ありがとうございます、フュルさん……お手伝いさせてもらって私も嬉しかったです。=Thanks, Miss Fur... I was happy to help. 
+こちらこそ、ありがとうございました。=You have my thanks as well.
+もしよかったら、今後もお手伝いに来てもよろしいでしょうか。=If it's allright, i can come help you in the future as well. 
+私は、特に構いませんが……=I wouldn't mind that...
+うふふ、それは良かったです。=Hehehe, I'm glad. 
+では、今度先輩がお客様とご一緒する時は、私がお手伝いに伺います。=In that case, the next time you are with the esteemed customer, i can come assist you as well.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/50/58/1/translation.txt:12]: i can > I can

**Pull request merged**